Libraries/LibCore/CNotifier.h

```diff
@@ -1,8 +1,9 @@
 #pragma once
 
 #include <AK/Function.h>
+#include "CObject.h"
 
-class CNotifier {
+class CNotifier : public CObject {
```

Comment:
Missing a `class_name()` override in this class now.
I wish they could appear automagically :|

Reply:
Well, we could mark it pure virtual. That would at least make it harder to forget them :)

Reply:
True, but only in the cases when you inherit directly from `CObject`..